.github/workflows/bench.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       with:
         role-to-assume: ${{ vars.ACTIONS_IAM_ROLE }}
         aws-region: ${{ vars.S3_REGION }}
+        role-duration-seconds: 21600
```

Comment:
The default assume role duration is 1 hour which is apparently not enough when the benchmark now takes 10x times.